science/hamlib/Portfile

```diff
@@ -1,25 +1,28 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        hamlib hamlib 3.3
+github.tarball_from releases
 
 name                hamlib
-version             3.0.1
 categories          science
 platforms           darwin
-maintainers         nomaintainer
+license             LGPL/GPL
+maintainers         {@ra1nb0w irh.it:rainbow} openmaintainer
 
 description         Ham Radio Control Libraries
 
 long_description    Flexible and portable shared libraries that offer a \
                     standardized API to control any radio oriented equipment \
                     through a computer interface.
 
-homepage            http://hamlib.org/
-master_sites        sourceforge
+homepage            http://www.hamlib.org/
```

Comment:
The `homepage` redirects to `https://hamlib.github.io`, so use that.

Reply:
I always prefer to use what the developer declare on `README` but if everything is on github probably can be use that.